sphinx/source/demo.rst

```diff
@@ -30,7 +30,9 @@ Given the above ``scenario.json`` this should create 2 nodes on the local machin
     :end-before: SNIPPET_END: create_network
     :dedent: 2
 
-Each transaction listed in the scenario is then sent to either the `primary` or a `follower` node. As the current implementation uses Raft for consensus, any ``Write`` transactions (which modify the KV) must be sent to the `primary`. Sending them to a `follower` will result in a ``TX_NOT_LEADER`` error response. ``Read`` transactions can be go to any node.
+Each transaction listed in the scenario is then sent to either the `primary` or a `backup` node. Any ``Write`` transactions (which modify the KV) must be processed by the `primary`. If a ``Write`` transaction is sent to a `backup` it will be forwarded to the `primary`.
+
+In the unlikely event that the `primary` has been changed by the time the transaction is forwarded, the transaction will result in a ``TX_NOT_PRIMARY`` error response. ``Read`` transactions can be sent to and processed by any node.
```

Comment:
I'm not convinced we need to mention this is our user-facing documentation. This is very unlikely to happen in practice and might cause more confusion than guidance.